server/.vscode/settings.json

```diff
@@ -0,0 +1,15 @@
+{
+    "cSpell.words": [
+        "Bson",
+        "invalidemail",
+        "invalidrole",
+        "Javalin",
+        "notanumber",
+        "OHMNET",
+        "sams",
+        "Servlet",
+        "sortorder",
+        "todos"
+    ],
+    "java.configuration.updateBuildConfiguration": "interactive"
+}
```

Comment:
Do we want to generate a .vscode folder and put these words in the spell checker?

Reply:
Isn't there already a `.vscode` folder with the `cSpell.words` entry?

Reply:
Maybe there isn't. If not, this will make that if/when we merge it in, and we should be good.